tests/e2e-core-common/pom.xml

```diff
@@ -43,6 +43,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Please create new proxy submodule in tests\integration\externalproperties\
and put the test there. Hopefully, more external properties would appear there